pkg/cloud_provider/file/file.go

```diff
@@ -474,13 +479,15 @@ func (manager *gcfsServiceManager) CreateBackup(ctx context.Context, obj *Servic
 	glog.V(4).Infof("Creating backup object %+v for the URI %v", *backupobj, backupUri)
 	opbackup, err := manager.backupService.Create(locationURI(obj.Project, region), backupobj).BackupId(backupName).Context(ctx).Do()
 	if err != nil {
-		return nil, fmt.Errorf("Create Backup operation failed: %v", err)
+		glog.Errorf("Create Backup operation failed")
+		return nil, err
 	}
 
 	glog.V(4).Infof("For backup uri %s, waiting for backup op %v to complete", backupUri, opbackup.Name)
 	err = manager.waitForOp(ctx, opbackup)
 	if err != nil {
-		return nil, fmt.Errorf("WaitFor CreateBackup op %s for source instance %v, backup uri: %v, operation failed: %v", opbackup.Name, backupSource, backupUri, err)
+		glog.Errorf("WaitFor CreateBackup op %s for source instance %v, backup uri: %v, operation failed", opbackup.Name, backupSource, backupUri)
```

Comment:
What's the reason for logging the errror here instead of adding more info and return it?